File: algorithms/0944.delete-columns-to-make-sorted/solution.rs
impl Solution {
    pub fn min_deletion_size(strs: Vec<String>) -> i32 {
        let mut count = 0;
        for col in 0..strs[0].len() {
            for row in 1..strs.len() {
                if strs[row - 1].chars().nth(col) > strs[row].chars().nth(col) {
                    count += 1;
                    break;
                }
            }
        }
        count
    }
}

File: algorithms/0944.delete-columns-to-make-sorted/solution.ts
function minDeletionSize(strs: string[]): number {
  let count = 0;
  for (let col = 0; col < strs[0].length; col++) {
    for (let row = 1; row < strs.length; row++) {
      if (strs[row - 1].split("")[col] > strs[row].split("")[col]) {
        count++;
        break;
      }
    }
  }
  return count;
}
